src/prr.rs

```diff
@@ -126,16 +129,28 @@ impl Prr {
         })
     }
 
+    /// Returns path to prr workdir
     fn workdir(&self) -> Result<PathBuf> {
-        match &self.config.prr.workdir {
+        let mut candidates = VecDeque::from([&self.config.prr.workdir]);
+        if let Some(lcfg) = &self.config.local {
+            candidates.push_front(&lcfg.workdir);
+        }
+
+        match candidates.into_iter().flatten().next() {
             Some(d) => {
                 if d.starts_with('~') {
-                    bail!("Workdir may not use '~' to denote home directory");
+                    bail!("Invalid workdir={d}: may not use '~'");
+                }
+
+                let p = Path::new(d).to_path_buf();
+                if !p.is_absolute() {
+                    bail!("Invalid workdir={d}: must be absolute path");
```

Comment:
For local config, I think it might be a good idea to allow relative paths, as long as they are relative to the config file itself (so that cwd of the process itself doesn't matter)

Reply:
Sure, makes sense. I'll fix it up tonight